modules/issue_tracker/ajax/EditIssue.php

```diff
@@ -388,6 +388,10 @@ function getComments($issueID)
 
     //looping by reference so can edit in place
     foreach ($unformattedComments as &$comment) {
+        $comment["dateAddedFormatted"] = date(
```

Comment:
I'm not talking about date formatting, although I believe both JS and PHP have a fairly easy ways to format dates without extra dependencies. 

I'm taking about treating this `$comments` array as data you return from an API which you want to have a nice structural format. i.e returning this:
```php
    $history = [
        '1' => [
            'user'    => $user,
            'date'    => $date,
            'changes' => ['field' => $key, 'value' => $value]
        ],
        ...
    ];
```
will allow to simply loop through the array an output data on the front-end, without unnecessary if statements and what not